addon/components/popover-on-element.js

```diff
@@ -1,122 +1,14 @@
-import Ember from 'ember';
-import TooltipAndPopoverComponent from 'ember-tooltips/components/tooltip-and-popover';
```

Comment:
I guess what I'm really trying to get to the bottom of is why we need the new `lazy-render-wrapper` component to achieve lazy loading of components.